public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -15,9 +15,9 @@ import { CloudWatchJsonData, CloudWatchSecureJsonData } from '../types';
 import { CancelablePromise, makePromiseCancelable } from 'app/core/utils/CancelablePromise';
 
 const authProviderOptions = [
+  { label: 'AWS SDK Default', value: 'default' },
```

Comment:
I would like somehow the UI to notify the users (apart from the logs) in case their configuration is deprecated. Now it does not appear any authenticator provider selected.
![Screenshot 2020-10-05 at 21 45 34](https://user-images.githubusercontent.com/1632407/95178669-61c3af00-07c8-11eb-8329-ffe9cdb16282.png)

Reply:
@papagian Who would implement the UI part?

Reply:
I can try